### ISSUE TITLE
NTR: Myocardial rupture (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8993]
New term request Myocardial rupture (elthat_110514025648): Heart; cardiac ventricular  UMLS: 0018813

**Comment by pnrobinson** [comment]
@MickeySegal
If this is the finding post-MI, then probably it would be better to call it LV rupture? https://www.ncbi.nlm.nih.gov/books/NBK559271/

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1494/
"Ruptures of the chordae tendinae or ventricle of the heart are rare cardiovascular complications"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001036
- label: Left ventricular rupture
- parents: [Abnormal left ventricle morphology (HP:0001711)](https://hpo.jax.org/app/browse/term/HP:0001711) | [Abnormal myocardium morphology (HP:0001637)](https://hpo.jax.org/app/browse/term/HP:0001637)
- pmid: PMID:20301667|PMID:32644697
- def: A full thickness tear or breakage of the myocardium of the left ventricle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8993
- synonyms:
